x-pack/plugin/core/build.gradle

```diff
@@ -118,7 +118,15 @@ tasks.named("thirdPartyAudit").configure {
           'org.apache.log.Logger',
           //commons-logging provided dependencies
           'javax.servlet.ServletContextEvent',
-          'javax.servlet.ServletContextListener'
+          'javax.servlet.ServletContextListener',
```

Comment:
I think some entries are duplicated?

Reply:
You're right! I've removed the superfluous entries.